plugins/woocommerce/src/Blocks/BlockTypes/Checkout.php

```diff
@@ -57,6 +58,63 @@ public function dequeue_woocommerce_core_scripts() {
 		wp_dequeue_style( 'select2' );
 	}
 
+	/**
+	 * Exposes settings exposed by the checkout block.
+	 */
+	public function register_settings() {
```

Comment:
Would be nice to limit this to only admin requests of REST as I don't (yet) think this should be a global option.